packages/amplify-graphql-conversation-transformer/src/resolvers/write-message-to-table-resolver-fn.js

```diff
@@ -0,0 +1,24 @@
+import { util } from '@aws-appsync/utils';
+import * as ddb from '@aws-appsync/utils/dynamodb';
+
+export function request(ctx) {
```

Comment:
as above, let's rename this and anything else that runs through substitution logic to indicate that it's not executable JS as-is.

Reply:
addressed in https://github.com/aws-amplify/amplify-category-api/commit/b05fbac4e071750d75b5c3523546ca0aa2245ed9